src/test/java/co/rsk/federate/btcreleaseclient/BtcReleaseClientTest.java

```diff
@@ -765,6 +768,129 @@ void onBestBlock_whenOnlySvpSpendTxWaitingForSignaturesIsAvailable_shouldAddSign
         );
     }
 
+    @Test
+    void onBestBlock_whenSvpSpendTxWaitingForSignaturesIsAvailableWithSignatureFromAnotherFederationMember_shouldSendAddSignature() throws Exception {
+        // Arrange
+        List<BtcECKey> federationPrivateKeys = TestUtils.getFederationPrivateKeys(9);
```

Comment:
```suggestion
        List<BtcECKey> federationKeys = TestUtils.getFederationPrivateKeys(9);
```

actually they are the pub keys **with** the private keys. The method name is not very accurate

Reply:
Updated